database/migrations/2021_07_08_111924_create_channels_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateChannelsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('channels', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->foreignId('user_id')->unique();
+            $table->text('fcm_token');
+            $table->timestamps();
+
+            $table->foreign('user_id')->references('id')->on('users');
```

Comment:
Isso é mais uma dica para escrever menos código. A linha pode ser simplificada como:

```suggestion
            $table->foreignId('user_id')->constrained();
```